<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebWid</class>
 <widget class="QMainWindow" name="WebWid">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>404</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>自考消息快递</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/img/icon.xpm</normaloff>:/img/icon.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Get the news from the Internet again.</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="icon">
     <iconset theme="face-smile">
      <normaloff/>
     </iconset>
    </property>
    <property name="shortcut">
     <string>Alt+R</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>661</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>News &amp; notice from Beijing Educational Examinations Authority for self-study students</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OKButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>I've read all of them.</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="shortcut">
     <string>Alt+O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="systrayButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>minimize to system tray</string>
    </property>
    <property name="text">
     <string>System Tray</string>
    </property>
    <property name="shortcut">
     <string>Alt+S</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>refreshButton</tabstop>
 </tabstops>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
